nestor_api/api/api_routes/sample.py

```diff
@@ -0,0 +1,11 @@
+"""Define the sample route."""
+from flask import Blueprint, Response, jsonify
+
+
+def register_routes(api: Blueprint) -> None:
+    """Register the /sample route."""
+
+    @api.route("/sample", methods=["GET"])
+    def sample() -> Response:  # pylint: disable=unused-variable
```

Comment:
It may be good to fix this warning as I am afraid we will have it for every route we add.
Try having a look at this --> https://github.com/jschaf/pylint-flask
(disclaimer: this is the first result I found, I did not try it 🙂 )

Reply:
Workaroudn seems to be to prefix with `_`